assets/scss/components/_forms.scss

```diff
@@ -0,0 +1,194 @@
+element inside the `label` element.
+
+A class of `checkbox` is __required__ on checkbox
+input elements.
+
+```html_example
+<label for="check1" class="label--minor chunk">
+	<input id="check1" class="checkbox" value="green" type="checkbox" />
+	Green
+</label>
+<label for="check2" class="label--minor chunk">
+	<input id="check2" class="checkbox" value="blue" type="checkbox" />
+	Blue
+</label>
+<label for="check3" class="label--minor chunk">
+	<input id="check3" class="checkbox" value="blue" type="checkbox" checked />
+	Already checked
+</label>
+```
+*/
+.checkbox {
+	cursor: pointer;
+	display: inline-block;
+	font-size: $font-size;
+	height: $font-size;
+	line-height: $line-height;
+	margin: 0 $space-half 0 0;
+
+	@include browser-lessThanIE(10) {
+		top: $space/2;
+	}
+}
+
+.label--minor {
+	font-weight: normal;
+}
+
+/*doc
+---
+title: Radios
+name: radio
+category: Forms
+---
+A label variant class of `label--minor` is
+used for side-by-side inputs and labels.
+
+For radios and checkboxes, we put the `input`
+element inside the `label` element.
+
+A class of `radio` is __required__ on radio input
+elements.
+
+```html_example
+<label for="radio1" class="label--minor chunk">
+	<input id="radio1" class="radio" name="radioExample" value="green" type="radio" />
+	Green
+</label>
+<label for="radio2" class="label--minor chunk">
+	<input id="radio2" class="radio" name="radioExample" value="red" type="radio" />
+	Red
+</label>
+```
+*/
+.radio {
+	cursor: pointer;
+	display: inline-block;
+	font-size: $font-size;
+	height: $font-size;
+	line-height: 1;
+	line-height: $line-height;
+	margin: 0 $space-half 0 0;
+
+	@include browser-lessThanIE(10) {
+		top: $space/2;
+	}
+}
+
+/*doc
+---
+title: Validation errors
+name: formValidation
+category: Forms
+---
+Field errors set explicitly with `field--error`.
+
+Error text is set explicitly with `text--error`.
+
+```html_example
+<div class="chunk">
+	<label for="test-textinput">First name</label>
+	<input class="field--error" id="test-textinput" type="text" value="" />
+	<p class="text--error">First name can't be empty</p>
+</div>
+```
+
+*/
+.field--error {
+	border-color: $C_attention !important;
+	&:focus {
+		border-color: $C_attention !important;
+	}
+}
+
+/*doc
+---
+title: Field reset
+name: fieldReset
+category: Forms
+---
+Reset default form styles with `.field--reset`.
+
+```html_example
+<div class="chunk">
+	<input class="field--reset" type="text" value="Reset text field" />
+</div>
+<div class="chunk">
+	<input class="field--reset" type="number" value="42" />
+</div>
+```
+*/
+input.field--reset,
+input.field--reset:hover,
+input.field--reset:focus,
+input.field--reset:active {
+	@include resetFieldStyles;
+}
+
+/*doc
+---
+title: Faux Input
+name: fauxInput
+category: Forms
+---
+Faux inputs visually mimic actual input elements.
+
+```html_example
+<div class="chunk">
+	<div class="fauxInput">I'm a faux input</div>
+</div>
+```
+*/
+.fauxInput {
```

Comment:
`fauxInput` is only used on `NumberInput` right now, but it could be used for similar things in the future